cmd/sonobuoy/app/delete.go

```diff
@@ -52,35 +53,59 @@ func NewCmdDelete() *cobra.Command {
 }
 
 func deleteSonobuoyRun(cmd *cobra.Command, args []string) {
+	s := spinner.New(spinner.CharSets[0], 100*time.Millisecond)
+	s.Color("red")
+
+	if runflags.wait > 0 {
+		s.Start()
+	}
 	cfg, err := deleteFlags.kubeconfig.Get()
 	if err != nil {
+		if deleteFlags.wait > 0 {
+			s.FinalMSG = "Status: Could not get kubernetes configuration!"
+			s.Stop()
+		}
 		errlog.LogError(errors.Wrap(err, "couldn't get kubernetes config"))
 		os.Exit(1)
 	}
 
 	sbc, err := getSonobuoyClient(cfg)
 	if err != nil {
+		if deleteFlags.wait > 0 {
+			s.Suffix = "Status: Could not create Sonobouy Client!"
+			s.Stop()
+		}
 		errlog.LogError(errors.Wrap(err, "could not create sonobuoy client"))
 		os.Exit(1)
 	}
 
 	client, err := sbc.Client()
 	if err != nil {
+		if deleteFlags.wait > 0 {
+			s.Stop()
```

Comment:
This pattern is repeated elsewhere, where on each path you check/stop the spinner.

The intuitive thing is after starting the spinner to just say `defer s.Stop()`, does that interact poorly with the rest of the UI?